site/content/docs/private-locations/checkly-agent-guide.md

```diff
@@ -167,6 +167,16 @@ If you lose track of which agent containers are using the old API key, you can u
 
 9) Click Save to close the dialog. You will now only see the new API key listed for the private location.
 
+## Trusing Enterprise TLS Certificates
+
+Some enterprise environments may use internal certificates for TLS connections. These certificates need to be trusted by the agent; otherwise, the agent's TLS connection to the Checkly platform will fail. 
+
+To configure the agent to trust the certificate, first copy the certificate as a PEM file onto the host. The Docker run command should then be updated to mount the certificate as a volume and pass the path to the certificate in the `NODE_EXTRA_CA_CERTS` environment variable. For a certificate stored at the path `~/certificate.pem`, the Docker run command will be:
```

Comment:
Should we also add the option on how to disable cert auth completely?

Reply:
I'm not so sure. Disabling certificate authentication is going to be insecure. I imagine it defeats the purpose of these enterprise network setups. Maybe it's useful for quickly trying out the agent without the root certificate, though.

Maybe something like:
> For running the agent without the root certificate, it's also possible to disable certificate authentication completely. This can be done by setting the environment variable `NODE_TLS_REJECT_UNAUTHORIZED=0`. Since this option completely disables certificate authentication, though, it's insecure and not recommended aside from testing purposes.

What do you think?

Reply:
I'd say we do not document this as it is a bad practice and we have not run into any customer needing this.